docs/src/pages/core-concepts/astro-components.md

```diff
@@ -184,6 +184,15 @@ const { greeting = 'Hello', name } = Astro.props;
 </div>
 ```
 
+You can then pass the component props like this:
+
+```astro
+---
+let firstName = "world!"
```

Comment:
```suggestion
// SomeOtherComponent.astro
import SomeComponent from "./SomeComponent.astro";
let firstName = "world!";
```

Could you add an import of the fictional component? Just for getting slightly closer to "copy and paste all this demo code and it works"